apps/generator/docs/deprecate-cli.sj-ag.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Deprecate Cli.js"
+weight: 170
+---
+
+# Migration Guide from “ag/asyncapi-generator” to AsyncAPI CLI
+
+## Overview
+With the introduction of the AsyncAPI CLI, the use of `Cli.js` for documentation generation in the AsyncAPI generator repository is being deprecated. This guide provides detailed instructions on how to transition from `ag` to the new AsyncAPI CLI.
+
+## Why Migrate?
+- **Enhanced Features:** The AsyncAPI CLI offers advanced features and improvements.
+- **Consistency:** Ensures consistent command usage across different environments.
+- **Support and Maintenance:** Future updates and support will focus on the AsyncAPI CLI.
+
+## Deprecated `ag/asyncapi-generator` Options and Their AsyncAPI CLI Equivalents
+Here is a list of `ag/asyncapi-generator` options and their equivalents in the AsyncAPI CLI:
+
+- **-d, --disable-hook [hooks...]**
+  - **AsyncAPI CLI Equivalent:** `asyncapi generate fromTemplate <ASYNCAPI> <TEMPLATE> --disable-hook <hookType>=<hookName>`
+
+- **--debug**
+  - **AsyncAPI CLI Equivalent:** `asyncapi generate fromTemplate <ASYNCAPI> <TEMPLATE> --debug`
+
+- **-i, --install**
+  - **AsyncAPI CLI Equivalent:** `asyncapi generate fromTemplate <ASYNCAPI> <TEMPLATE> --install`
+
+- **-n, --no-overwrite <glob>**
+  - **AsyncAPI CLI Equivalent:** `asyncapi generate fromTemplate <ASYNCAPI> <TEMPLATE> --no-overwrite <glob>`
+
+- **-o, --output <outputDir>**
+  - **AsyncAPI CLI Equivalent:** `asyncapi generate fromTemplate <ASYNCAPI> <TEMPLATE> --output <outputDir>`
+
+- **-p, --param <name=value>**
+  - **AsyncAPI CLI Equivalent:** `asyncapi generate fromTemplate <ASYNCAPI> <TEMPLATE> --param <name=value>`
+
+- **--force-write**
+  - **AsyncAPI CLI Equivalent:** `asyncapi generate fromTemplate <ASYNCAPI> <TEMPLATE> --force-write`
+
+- **--watch-template**
+  - **AsyncAPI CLI Equivalent:** `asyncapi generate fromTemplate <ASYNCAPI> <TEMPLATE> --watch`
+
+- **--map-base-url <url:folder>**
+  - **AsyncAPI CLI Equivalent:** `asyncapi generate fromTemplate <ASYNCAPI> <TEMPLATE> --map-base-url <url:folder>`
+
+## Migration Steps
+
+### 1. Install AsyncAPI CLI
+First, ensure you have the AsyncAPI CLI installed:
```

Comment:
please add note that there are different methods that do not require setup of node environment -> https://www.asyncapi.com/docs/tools/cli/installation

the best probably would be to use https://github.com/asyncapi/website/blob/master/assets/docs/fragments/cli-installation.md fragment - this is how to use it -> https://github.com/asyncapi/website/blob/master/markdown/docs/tutorials/generate-code.md?plain=1#L32